doc/api/process.md

```diff
@@ -1824,6 +1890,14 @@ cases:
   code will be `128` + `6`, or `134`.
 
 
+## process.workerData
+<!-- YAML
+added: REPLACEME
+-->
+
+An arbitrary JavaScript object that contains a clone of the data passed
```

Comment:
"contains a clone" could use clarifying - for example what happens to functions on the object.

Reply:
Yeah, I’m linking the HTML structured clone algorithm here as well